### PR TITLE
Add flake8 config and requirement

### PR DESCRIPTION
**Pull request by kennydo**
This builds on top of the PR that puts Flask app instantiation into a
function, so that should be merged first.

I piped the output of the `pip freeze` into requirements, so that's why things were sorted and shuffled around.

Tested by deleting my local `sqlite.db`, running `run_server.py`, verifying sqlite still got created, and then verifying that hitting `http://0.0.0.0:5000/` gave "Hello, World".

### CODE REVIEW

File: my_service/app/views/__init__.py [hunk 1]
```diff
@@ -1 +0,0 @@
-from .Example.example import Example
```
Comment (21echoes) [inline my_service/app/views/__init__.py]: sadly, afaict Jinja needs this to be here

**Comment by 21echoes** [comment]
one small comment. otherwise, lgtm

File: my_service/app/views/__init__.py (hunk 1 above)
Comment (kennydo) [inline my_service/app/views/__init__.py]: Strange, I can get the "Hello, World" example page without any issue on http://0.0.0.0:5000/. 

When does Jinja need this here?

Comment (21echoes) [inline my_service/app/views/__init__.py]: you need it for partials. see how we enable it https://github.com/Patreon/service-starter-kit-python/blob/master/my_service/app/views/configure_jinja.py#L25 and example usage https://github.com/Patreon/beagle/blob/master/beagle/app/views/Dashboard/dashboard.html#L104

Comment (kennydo) [inline my_service/app/views/__init__.py]: Oof, bummer. Adding back.

**Pull request merged**